draft-ietf-httpbis-connect-tcp.md

```diff
@@ -128,9 +128,9 @@ This section discusses some behaviors that are permitted or recommended in order
 
 ## Latency optimizations
 
-When using this specification in HTTP/2 or HTTP/3, clients MAY start sending TCP stream content without waiting for an HTTP response.  Proxies MUST buffer this "false start" content until the TCP stream becomes writable, and discard it if the TCP connection fails.  (This "false start" behavior is not permitted in HTTP/1.1 because it would prevent reuse of the connection after an error response such as 407 (Proxy Authentication Required).)
+When using this specification in HTTP/2 or HTTP/3, clients MAY start sending TCP stream content optimistically, without waiting for an HTTP response, up to the proxy's initial flow-control window size ({{!RFC9113, Section 6.9.2}})({{!RFC9000, Section 4.1}}).  Proxies MUST buffer this "optimistic" content until the TCP stream becomes writable, and discard it if the TCP connection fails.  (This "optimistic" behavior is not permitted in HTTP/1.1 because it would prevent reuse of the connection after an error response such as "407 (Proxy Authentication Required)".)
```

Comment:
Any reason to restrict to the initial flow control window? The client must respect flow control limits, but if the server wants to open it up further before sending the response, I'm not sure why the spec should restrict that?

Reply:
Thanks, that's a bug in my text.  Fixed!

Reply:
Thanks, that's a bug in my text.  Fixed!